src/console.cpp

```diff
@@ -300,24 +300,24 @@ void IConsoleCmdExec(const char *cmdstr, const uint recurse_count)
 	bool longtoken = false;
 	bool foundtoken = false;
 
-	if (cmdstr[0] == '#') return; // comments
+	if (command_string[0] == '#') return; // comments
 
-	for (cmdptr = cmdstr; *cmdptr != '\0'; cmdptr++) {
+	for (cmdptr = command_string.c_str(); *cmdptr != '\0'; cmdptr++) {
```

Comment:
Can this for loop (& the one below) be cleaned up a bit more?

Reply:
It can probably be cleaned up a bit more, though arguably rethinking/redesigning the whole function with std::string(_view) functions would yield a better result than doing some tiny improvements.

And that's in my opinion out of scope for this PR, as I just want to not add more `.c_str()`s when changing other callers of `IConsoleCmdExec` to use std::string, to get rid of `strecpy`.